addons/api/addon/utils/indexed-db-query.js

```diff
@@ -223,6 +223,15 @@ const getKey = (key) => {
   }
 };
 
+const cleanValues = (values) => {
+  return values.reduce((result, obj) => {
+    // eslint-disable-next-line no-unused-vars
+    const [key, value] = Object.entries(obj)[0];
```

Comment:
we can use an underscore to replace `key` so that we don't have to disable lint rule
```suggestion
    const [_, value] = Object.entries(obj)[0];
```